client/src/hooks/useMockedResource.js

```diff
@@ -0,0 +1,14 @@
+import { useState, useEffect, useContext } from "react";
+
+const mockData = require("../helpers/mockData.json");
```

Comment:
This file is not in the repo, so this line breaks the client

Reply:
Indeed. I'm not sure it makes sense to have a dedicated hook for mocking anyway, so I just deleted the file